src/main/kotlin/com/terraformation/backend/accelerator/api/ProjectVotesController.kt

```diff
@@ -0,0 +1,127 @@
+import com.terraformation.backend.db.default_schema.UserId
+import io.swagger.v3.oas.annotations.Operation
+import io.swagger.v3.oas.annotations.media.Schema
+import org.springframework.web.bind.annotation.DeleteMapping
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.PutMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+
+@AcceleratorEndpoint
+@RequestMapping("/api/v1/accelerator/projects/{projectId}/votes")
+@RestController
+class ProjectVotesController() {
+  @ApiResponse200
+  @ApiResponse403
+  @ApiResponse404
+  @GetMapping
+  @Operation(
+      summary = "Gets vote selections for a single project.",
+      description =
+          "List every vote selection for this project, organized by phases. Each phase will " +
+              "contain a list of eligible voters and their selections. ")
+  fun getProjectVotes(
+      @PathVariable("projectId") projectId: ProjectId,
+  ): GetProjectVotesResponsePayload {
+    return GetProjectVotesResponsePayload(projectId, "testProject", emptyList())
+  }
+
+  @ApiResponse200
+  @ApiResponse403
+  @ApiResponse404
+  @PutMapping
+  @Operation(
+      summary = "Upserts vote selections for a single project.",
+      description =
+          "Update the user's vote for the project phase. If the (user, project, phase) does not " +
+              "exist, a new entry is created. Setting a `voteOption` to `null` removes the vote.")
+  fun upsertProjectVotes(
+      @PathVariable("projectId") projectId: ProjectId,
+      @RequestBody payload: UpsertProjectVotesRequestPayload,
+  ): UpsertProjectVotesResponsePayload {
+    return UpsertProjectVotesResponsePayload(projectId, payload.phase, emptyList())
+  }
+
+  @ApiResponse200
+  @ApiResponse400
+  @ApiResponse403
+  @ApiResponse404
+  @DeleteMapping
+  @Operation(
+      summary = "Remove one or more voters from the project/phase.",
+      description =
+          "Remove the voters from the project phase, making them ineligible from voting. This is " +
+              "different from undoing a vote (by setting the `voteOption` to `null`). To remove " +
+              "voters from the entire project phase, set `userId` to `null`, and set " +
+              "`phaseDelete` to `true`")
+  fun deleteProjectVotes(
+      @PathVariable("projectId") projectId: ProjectId,
+      @RequestBody payload: DeleteProjectVotesRequestPayload,
+  ): SimpleSuccessResponsePayload {
+    return SimpleSuccessResponsePayload()
+  }
+}
+
+data class VoteSelection(
+    val userId: UserId,
+    @Schema(
+        description =
+            "The vote the user has selected. Can be yes/no/conditional or `null` if " +
+                "a vote is not yet selected.")
+    val voteOption: VoteOption? = null,
```

Comment:
@tommylau523 if the user selects 'Conditional', the designs show a text entry where user expands on decision. We would need to store/return this text data.